classes/author.rb

```diff
@@ -0,0 +1,16 @@
+class Author
+  attr_accessor :items
+  attr_reader :first_name, :last_name
+
+  def initialize(first_name, last_name)
+    @id = Random.rand(1..100)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
+    item.add_author(self)
+  end
```

Comment:
- [ ] in this method the last line should be `item.author = self` similar to what was done in the genre and label class.